ProcessMaker/processmaker
# FOUR-13969: Allow user to decide what to do with the Screen Interstitial when importing assets

## Issue & Reproduction Steps
When following the guided template process, it has been noticed that the interstitial screen does show within the TemplateAssets check and it's been duplicated multiple times. 


Interstitial screens should be part of the initial existing assets check, so the user can decide if they’ll be updated, replaced, or if a new version should be created

## Solution
- Allow Screen Interstitial to display within the TemplateAssets table.

## How to Test
- Navigate to Processes
- Navigate to Guided Templates
- Select the Guided Invoice Approval
- Follow all the steps
Please ensure that the user has the option of what to do with the Interstitial Screen.
Please make sure that each selection works as expected. (Update | Keep Previous | Duplicate)

![Screenshot 2024-01-31 at 5 22 10 PM](https://github.com/ProcessMaker/processmaker/assets/5769433/7e1eb890-4679-4d97-a4f4-946d8a9649c4)

## Related Tickets & Packages
- Ticket: [FOUR-13969](https://processmaker.atlassian.net/browse/FOUR-13969)
ci:next

## Code Review Checklist


[FOUR-13969]: https://processmaker.atlassian.net/browse/FOUR-13969?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
